### PR TITLE
Added bad-value-for-attribute-src-on-element-iframe-must-be-non-empty

### PR DESCRIPTION
**Pull request by rujaun**

### CODE REVIEW

File: compatibility/html/bad-value-for-attribute-src-on-element-iframe-must-be-non-empty.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Empty src attribute was found in an iframe. According to the W3C iframe src attributes may not be empty. The combination of an empty iframe loaded inside a secure web page (SSL/HTTPS) could possibly make browsers display 'This page has insecure content' or similar error and might block the iframe content all together.
+
+`<iframe src=""></iframe>`
```
Comment (Johanndutoit) [inline compatibility/html/bad-value-for-attribute-src-on-element-iframe-must-be-non-empty.md]: Think we should just update this to show a code block like - https://github.com/passmarked/suite/commit/2c26072c269a6c60c9e61b77c7c1292922cac724#diff-bb8e40a0124e6ccb096bc5a6a78ecbebR3 - which will format the HTML nicely too.

File: compatibility/html/bad-value-for-attribute-src-on-element-iframe-must-be-non-empty.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# How do I fix this ?
+
+Place `about:blank` inside the `src` attribute:
+`<iframe src='about:blank'></iframe>`
```
Comment (Johanndutoit) [inline compatibility/html/bad-value-for-attribute-src-on-element-iframe-must-be-non-empty.md]: Thinking same for this line like https://github.com/passmarked/suite/commit/2c26072c269a6c60c9e61b77c7c1292922cac724#diff-bb8e40a0124e6ccb096bc5a6a78ecbebR3

Comment (rujaun) [inline compatibility/html/bad-value-for-attribute-src-on-element-iframe-must-be-non-empty.md]: Agreed. Will also change single quotes to double

**Comment by Johanndutoit** [comment]
+1 from me, just the code blocks but didn't spot a error. More reviews to follow.

**Comment by Johanndutoit** [comment]
Looks good to me :)

**Pull request merged**